src/mappings/Controller/mapping.ts

```diff
@@ -136,6 +164,36 @@ export function handleRegisterScheme(event: RegisterScheme): void {
     return;
   }
 
+  let dao = DAO.load(avatar.toHex());
+  let controllerScheme = ControllerScheme.load(crypto.keccak256(concat(avatar, event.params._scheme)).toHex());
+  if (dao != null && controllerScheme != null) {
+    dao.numberOfQueuedProposalsUnregistered = dao.numberOfQueuedProposalsUnregistered.minus(
```

Comment:
is there a scenario that this can be negative?

Reply:
Yes if the scheme is re-registered